packages/frontend/src/features/catalog/components/CatalogTableListItem.tsx

```diff
@@ -124,16 +131,10 @@ export const CatalogTableListItem: FC<React.PropsWithChildren<Props>> = ({
                         }}
                     >
                         <MantineLinkButton
-                            size="xs"
+                            size="sm"
                             href={url}
                             target="_blank"
                             compact
-                            rightIcon={
-                                <MantineIcon
-                                    size="sm"
-                                    icon={IconExternalLink}
-                                />
-                            }
                             sx={(theme) => ({
```

Comment:
I asked for this because im rethinking what happens when you click on a row: 

https://lightdash.slack.com/archives/C0741H6LVNY/p1717074072132999


For now, im just prioritising decluttering, but I see your point! Gonna test my above assumption next week!